docs/rules/no-parent-imports.md

```diff
@@ -0,0 +1,35 @@
+# no-parent-imports
+
+This will disallow imports that contain ../.. 
+
+### Rule Details
+
+When we want to go to far up to import a module, we should use an absolute import.
+
+### Examples of **incorrect** code for this rule:
```

Comment:
Here I think you meant  `correct` and down there the `incorrect` examples

Reply:
Thanks!
https://github.com/pmedianetwork/eslint-config/pull/64/commits/59553db45ded52fde5f77a5eacaf7998023fa335